src/application-base/ApplicationBase.jsx

```diff
@@ -73,10 +73,14 @@ const propTypes = {
    * before continuing with the unload event, even if NavigationPrompts are present.
    */
   unloadPromptIsDisabled: PropTypes.bool,
+  /**
+   * A Callback for errrors caught by the Error Boundary.
+   */
+  onError: PropTypes.func,
```

Comment:
We need to be way more descriptive with this prop.

Reply:
Should this apply to all Application error boundaries when it's set here? Or should it be available in from terra-application to propagate to them?